### PR TITLE
Added OntoFlow KPA ontology

### PR DESCRIPTION
**Pull request by xAlessandroC**
Add OntoFlow KPA ontology to repo

### CODE REVIEW

File: KPA/kpa.ttl [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix emmo: <http://emmo.info/emmo#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@base <http://open-model.eu/ontoflow/kpa> .
+
+<http://open-model.eu/ontoflow/kpa> rdf:type owl:Ontology ;
+                                     owl:imports <http://emmo.info/emmo#1.0.0-beta4> .
```
Comment (jesper-friis) [inline KPA/kpa.ttl:11]:
```suggestion
                                     owl:imports <https://w3id.org/emmo/1.0.0-rc1> .
```

File: KPA/kpa.ttl [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+#################################################################
+#    Object Properties
+#################################################################
+
+###  http://open-model.eu/ontoflow/kpa#hasKPA
+:hasKPA rdf:type owl:ObjectProperty .
```
Comment (jesper-friis) [inline KPA/kpa.ttl:18]: Would be good to add elucidation and range (:KPA). 
Should probably be a subproperty of emmo:hasProperty.

File: KPA/kpa.ttl [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+
+#################################################################
+#    Data properties
+#################################################################
+
+###  http://open-model.eu/ontoflow/kpa#KPAValue
+:KPAValue rdf:type owl:DatatypeProperty .
```
Comment (jesper-friis) [inline KPA/kpa.ttl:26]: Would be good to add elucidation.
Should probably be a subproperty of emmo:hasDataValue.

File: KPA/kpa.ttl [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+
+#################################################################
+#    Classes
+#################################################################
+
+###  http://open-model.eu/ontoflow/kpa#Accuracy
+:Accuracy rdf:type owl:Class ;
```
Comment (jesper-friis) [inline KPA/kpa.ttl:34]: Please add elucidation.

File: KPA/kpa.ttl [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+
+###  http://open-model.eu/ontoflow/kpa#BoolOpenSource
+:BoolOpenSource rdf:type owl:Class ;
+                rdfs:subClassOf :OpenSource ;
+                rdfs:comment "A performance attribute indicating if the software is open source or not. Possible values are true and false"@en .
+
+
+###  http://open-model.eu/ontoflow/kpa#KPA
+:KPA rdf:type owl:Class ;
```
Comment (jesper-friis) [inline KPA/kpa.ttl:46]: elucidation

File: KPA/kpa.ttl [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+###  http://open-model.eu/ontoflow/kpa#OrdinalAccuracy
+:OrdinalAccuracy rdf:type owl:Class ;
+                 rdfs:subClassOf emmo:EMMO_50ea1ec5_f157_41b0_b46b_a9032f17ca10 ,
+                                 :Accuracy ;
+                 rdfs:comment "A performance attribute indicating the typical accuracy of a simulation tool. This takes the values of  High, Medium and Low"@en .
+
+
+###  http://open-model.eu/ontoflow/kpa#SimulationTime
+:SimulationTime rdf:type owl:Class ;
```
Comment (jesper-friis) [inline KPA/kpa.ttl:81]: elucidation

File: KPA/kpa.ttl [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+
+###  http://open-model.eu/ontoflow/kpa#NumericalSimulationTime
+:NumericalSimulationTime rdf:type owl:Class ;
+                         rdfs:subClassOf :SimulationTime ;
+                         rdfs:comment "A performance attribute indicating the typical simulation time, in seconds."@en .
+
+
+###  http://open-model.eu/ontoflow/kpa#OpenSource
+:OpenSource rdf:type owl:Class ;
```
Comment (jesper-friis) [inline KPA/kpa.ttl:69]: elucidation

File: KPA/kpa.ttl [hunk 8]
```diff
@@ -0,0 +1,85 @@
+@prefix : <http://open-model.eu/ontoflow/kpa#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix emmo: <http://emmo.info/emmo#> .
```
Comment (jesper-friis) [inline KPA/kpa.ttl:6]:
```suggestion
@prefix emmo: <https://w3id.org/emmo#> .
```

**Review by jesper-friis — COMMENTED** [review]
Added some suggestions.

File: KPA/kpa.ttl (hunk 8 above)
Comment (xAlessandroC) [reply KPA/kpa.ttl:6]: I think this change may be disruptive at the moment. Most of the examples we have, and also the ss3 ontology, use the URL `emmo.info`. How can we handle this? Can we postpone this change until after most application ontologies have been updated?

**Comment by xAlessandroC** [comment]
Added elucidations. I've also commented on one of your suggestions @jesper-friis (please see above)

File: KPA/kpa.ttl (hunk 8 above)
Comment (jesper-friis) [reply KPA/kpa.ttl:6]: How disruptive will it be for you?
It is something we have to change sooner or later. Isn't it easier to fix it now? But if it is very disruptive for you it is also ok for me to wait.

**Review by jesper-friis — APPROVED** [review]
Approved, but please consider whether we should change to w3id already now.

**Pull request merged**